cloudevents-sdk-warp/README.md

```diff
@@ -0,0 +1,110 @@
+# Cloud Event Support For Warp
+
+This crate provides [cloud event](https://github.com/cloudevents/sdk-rust) support for [warp server](https://github.com/seanmonstar/warp).
+
+This implementation is heavily inspired by [cloudevents-sdk-actix-web](https://github.com/cloudevents/sdk-rust/tree/master/cloudevents-sdk-actix-web)
+implementation.
+
+## Usage examples
+
+Basically event can be extracted from request or returned as response
+
+Extracting cloud event from a request and returning it back to sender:
+
+```rust
+use cloudevents_sdk_warp::{filter, reply};
+use warp::Filter;
+
+//  cargo run --example example_request
+#[tokio::main]
+async fn main() {
+    let routes = warp::any()
+        // extracting event from request
+        .and(filter::to_event())
+        // returning event back
+        .map(|event| reply::from_event(event));
+
+    warp::serve(routes).run(([127, 0, 0, 1], 3030)).await;
+}
+```
+
+executing `http` request:
```

Comment:
Maybe replace this with a `curl` command? (run it with `-v`, so you'll get the full response headers too)